src/encoded/schemas/file.json

```diff
@@ -417,17 +417,12 @@
             "type": "string",
             "linkTo": "pipeline"
         },
```

Comment:
@vsmalladi Do we still want "pipeline" here? We can now get to it via `step_run.workflow_run.pipeline`, though it looks like analysis_step_run does not require "workflow_run" at the moment.